src/core/Akka.Tests/Routing/TailChoppingSpec.cs

```diff
@@ -127,8 +127,8 @@ public void Tail_chopping_group_router_must_return_response_from_second_actor_af
         [Fact]
         public void Tail_chopping_group_router_must_throw_exception_if_no_result_will_arrive_within_the_given_time()
         {
-            var actor1 = Sys.ActorOf(Props.Create(() => new TailChopTestActor(500.Milliseconds())), "Actor3");
-            var actor2 = Sys.ActorOf(Props.Create(() => new TailChopTestActor(500.Milliseconds())), "Actor4");
+            var actor1 = Sys.ActorOf(Props.Create(() => new TailChopTestActor(1500.Milliseconds())), "Actor3");
```

Comment:
@alexvaluyskiy that's what commit histories are for - in this case the issue was just that the delay `500ms` and the timeout `300ms` were sufficiently close together that a busy system could receive the unwanted "ack" message before the delay. Nothing was technically wrong with the spec or the code it was testing otherwise.